api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java

```diff
@@ -247,17 +255,25 @@ public void setUp() {
 
     String createdDate = Date.fromYearMonthDay(1988, 12, 26).toString();
 
-    DataprocConfig dataprocConfig =
+    dataprocConfig =
         new DataprocConfig()
             .numberOfWorkers(0)
             .masterMachineType("n1-standard-4")
             .masterDiskSize(50);
 
-    LinkedTreeMap<String, Object> dataProcConfigObj = new LinkedTreeMap<>();
-    dataProcConfigObj.put("cloudService", "DATAPROC");
-    dataProcConfigObj.put("numberOfWorkers", 0);
-    dataProcConfigObj.put("masterMachineType", "n1-standard-4");
-    dataProcConfigObj.put("masterDiskSize", 50.0);
+    dataprocConfigObj = new LinkedTreeMap<>();
```

Comment:
wasn't sure if this is what you had in mind but I could only use LeonardoMapper to map `dataprocConfigObj` to `dataprocConfig`. Not as ideal as going the other direction but I also don't have that code implemented right now since we're not doing Runtime -> Leonardo mapping atm.